stdlib/3/macurl2path.pyi

```diff
@@ -0,0 +1,10 @@
+
+import sys
+
+from typing import AnyStr, Union
+
+# Is this correct? It seems to be for windows
+if sys.version_info < (3, 7):
+    def url2pathname(pathname: AnyStr) -> AnyStr: ...
```

Comment:
I think it just accepts str.

Reply:
Yeah, it does. I didn't check at first and just followed urllib's types for these functions, but that was a mistake